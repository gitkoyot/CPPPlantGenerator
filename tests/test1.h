#ifndef TEST_1_H
#define TEST_1_H

class FirstClass
{
    int firstField;
    bool secondField;
    char* firstMethod();
    long* secondMethod(int firstParameter);
};



struct FirstStructure 
{
    int firstField;
    bool secondField;
    char* firstMethod();
    long* secondMethod(int firstParameter);
};



#endif
