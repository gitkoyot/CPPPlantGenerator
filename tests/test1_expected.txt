@startuml
class FirstClass{
-firstField   :   int
-secondField   :   bool
-char * firstMethod ( )
-long * secondMethod ( firstParameter   :   int)
}
class FirstStructure{
+firstField   :   int
+secondField   :   bool
+char * firstMethod ( )
+long * secondMethod ( firstParameter   :   int)
}
@enduml

